### DISCUSSION TITLE
make_NCA()

### DISCUSSION THREAD

**Discussion by livPKPD** [LAPKB/Pmetrics#97] [General]
Hi again (apologies!)
I am trying to run make_NCA() or makeNCA(). The legacy version does not run as it wants 'mdata.x', which does not seem to be an element of PMload(x) anymore?

So I have been trying to get make_NCA(x, postPred=T) to run. I get this error: 
> res<-PM_load(71)
> make_NCA(
+   res$post$data,
+   postPred = T)
_Error in make_NCA(x$post$data, postPred = T) : 
  You must specify a PM_result object._
I thought I had done that with x<- PM_load(71)??

Then I tried coding it as: 
> nca71<- x$nca(postPred=T, icen="mean", start=144, end=168, outeq=1)
But I get: 
_Error in conv_post(post = post, mdata = mdata, input = input, outeq = outeq,  : 
  object 'doseTime' not found_

Any ideas? Apologies for incessantly bugging you!

**Comment by Siel** [comment]
Hi @livPKPD `res$post$data` if of type PMpost and not PM_result. res on the other hand is PM_result.

Regarding the second error, I need the data and model (or the fited object in rData format) to try to reproduce it by myself

Greetings

**Comment by Siel** [comment]
I found the source of the error with make_NCA, will report back when it's fixed

**Comment by Siel** [comment]
Hi @livPKPD, please install Version2 again. The issue should be fixed
